frontend/src/Activity/Queue/QueueStatus.js

```diff
@@ -154,8 +154,8 @@ QueueStatus.propTypes = {
 };
 
 QueueStatus.defaultProps = {
-  trackedDownloadStatus: translate('Ok'),
-  trackedDownloadState: translate('Downloading'),
+  trackedDownloadStatus: 'Ok',
+  trackedDownloadState: 'Downloading',
```

Comment:
Looks like these should be

```suggestion
  trackedDownloadStatus: 'ok',
  trackedDownloadState: 'downloading',
```